src/main/scala/rps/Game.scala

```diff
@@ -22,25 +22,22 @@ object Game {
   }
 
   def getMoveName(move: Move.Move): String = move match {
-    case (Move.Rock)    => "Rock"
-    case (Move.Paper)   => "Paper"
-    case (Move.Scissor) => "Scissor"
+    case Move.Rock    => "Rock"
+    case Move.Paper   => "Paper"
+    case Move.Scissor => "Scissor"
   }
 
-  def getGameResult(a: Move.Move, b: Move.Move): Result.Result = {
-    if (a == b) {
-      Result.Tie
-    } else
-      (a, b) match {
-        case (Move.Rock, Move.Paper)    => Result.Won
-        case (Move.Paper, Move.Scissor) => Result.Won
-        case (Move.Scissor, Move.Rock)  => Result.Won
-        case _                          => Result.Lost
-      }
-  }
+  def getGameResult(a: Move.Move, b: Move.Move): Result.Result =
+    (a, b) match {
+      case (a, b) if a == b => Result.Tie
+      case (Move.Rock, Move.Paper) | (Move.Paper, Move.Scissor) |
+          (Move.Scissor, Move.Rock) =>
+        Result.Won
+      case _ => Result.Lost
```

Comment:
super minor trick for readability: you can import `Move._` into the method scope, so that you can use the names directly. Also for readability, I would give significant names to `a`, `b`:

```scala
def getGameResult(computerMove: Move.Move, userMove: Move.Move): Result.Result = {
  import Move._
  (computerMove, userMove) match {
    case (a, b) if a == b => Result.Tie
    case (Rock, Paper) | (Paper, Scissor) | (Move.Scissor, Move.Rock) => Result.Won
    case _ => Result.Lost
  }
}
```